pkgs/development/python-modules/nocturne/default.nix

```
@@ -0,0 +1,61 @@
+{ buildPythonPackage
+, cmake
+, fetchFromGitHub
+, gtest
+, hydra-core
+, lib
+, nlohmann_json
+, pybind11
+, PyVirtualDisplay
+, sfml
+, substituteAll
+}:
+
+buildPythonPackage rec {
+  pname = "nocturne";
+  version = "unstable-2023-01-02";
```

https://github.com/facebookresearch/nocturne/commit/ae0a4e361457caf6b7e397675cc86f46161405ed

```suggestion
  version = "unstable-2022-10-15";
```

---

good point!